charts/teku/ci/default-values.yaml

```diff
@@ -1 +1,3 @@
 # Leave empty so that CT tests with default values
+extraArgs:
+  - --ee-endpoint: http://localhost:8551
```

Comment:
should be

``` 
extraArgs:
  - --ee-endpoint=http://localhost:8551
```

Reply:
Fixed. 
However looks like erigon also failed.

Reply:
Seems like the rpcd container is crashing on erigon:

```
2022-08-25T20:48:10.1792889Z [EROR] [08-25|20:46:38.995] Could not connect to DB                  err="mdbx_env_open: resource temporarily unavailable, label: chaindata, trace: [kv_mdbx.go:220 config.go:287 main.go:19 command.go:872 command.go:990 command.go:918 command.go:911 main.go:38 proc.go:250 asm_amd64.s:1571]"
```

Reply:
Yes, I can't get it to run anywhere. Their release seems to be broken. 

Hoping for a fix soon.

Reply:
The issue was caused by the fact that erigon isn't sharing process memory among different pods. With the latest release I have enabled shared memory among the erigon pods. And then I had issue where port 8545 was binding to erigon container instead of erigon-rpcd container. (Due to the shared memory among pods). So I had to disable http on erigon, and now it looks like it is fixed.

Related issues: 
https://github.com/dappnode/DAppNodePackage-Erigon/issues/34#issuecomment-1069744546
https://github.com/ledgerwatch/erigon/issues/4482